vm-setup/Vagrantfile.init

```diff
@@ -0,0 +1,8 @@
+Vagrant.configure("2") do |config|
+  config.vm.synced_folder ".", "/vagrant", type: "rsync"
+  config.vm.provider "virtualbox" do |vb|
+   vb.memory = "4096"
```

Comment:
Ok, so are we sure that 4G is enough? I.e. the CPU count of 2 is the deciding factor to make things stable?

Reply:
No we are not, that's why I kept 6 GB in the default Vagrant file